Windows build broke without the sed hack

v0.4.6 builds my project successfully on Windows: https://github.com/athenianco/ghid/actions/runs/4701826230/jobs/8338345932#step:4:80

v0.4.7 fails: https://github.com/athenianco/ghid/actions/runs/4701941867/jobs/8338644153#step:4:82

```
go build -mod=mod -buildmode=c-shared -o ghid_go.pyd .
cmd had error: exit status 1  output:
go: finding module for package github.com/go-python/gopy/gopyh
go: found github.com/go-python/gopy/gopyh in github.com/go-python/gopy v0.4.7
# github.com/athenianco/ghid/python/ghid
C:\hostedtoolcache\windows\go\1.20.3\x64\pkg\tool\windows_amd64\link.exe: running gcc failed: exit status 1
c:/programdata/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lpython311
collect2.exe: error: ld returned 1 exit status
```

--- a/bind/utils.go
+++ b/bind/utils.go
@@ -186,11 +186,10 @@ else:
 	raw.LibDir = filepath.ToSlash(raw.LibDir)
 
 	// on windows these can be empty -- use include dir which is usu good
+	// replace suffix case insensitive 'include' with 'libs'
 	if raw.LibDir == "" && raw.IncDir != "" {
-		raw.LibDir = raw.IncDir
-		if strings.HasSuffix(raw.LibDir, "include") {
-			raw.LibDir = raw.LibDir[:len(raw.LibDir)-len("include")] + "libs"
-		}
+		regexInc := regexp.MustCompile(`(?i)\binclude$`)
+		raw.LibDir = regexInc.ReplaceAllString(raw.IncDir, "libs")
 		fmt.Printf("no LibDir -- copy from IncDir: %s\n", raw.LibDir)
 	}
 

